frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "bnc-notify": "^1.5.1",
```

Comment:
Fixed

Reply:
A couple of notes:

1. Sending to an ENS name no longer causes an error, but it does not resolve and display the ENS name on the send history page. Is there a reason why it would be difficult to do that? If so we can pull it into a separate issue and ship this feature without it, but if possible it would be nice to resolve the names.
2. Sending to CNS names still results in an error similar to the one I was seeing with ENS before your fix.

Reply:
CNS should be working now, and I think the ENS name needs to be pulled into a separate issue because we have a method to lookup the ENS address, but  I couldn't get it to work even when hard coding known addresses.

![Screenshot from 2023-04-26 10-56-00](https://user-images.githubusercontent.com/12705146/234616226-ef7582de-e8bf-4c53-9b36-7144195d8664.png)

Reply:
can you expand on what the issue with ENS was? seems weird that CNS would work but not ENS

Reply:
Per standup yesterday let's make sure we're always using the `MAINNET_PROVIDER` for ENS parsing and see if that resolves the issue

Reply:
This fixed the issue

Reply:
Awesome. Leaving this unresolved for @apbendi since he raised a few points in this thread, so I'll let him verify the resolutions

Reply:
Name resolution seems to be working on the send history table. Thanks @alexkeating.